examples/balance/login/login.js

```diff
@@ -0,0 +1,55 @@
+const url = new URL(window.location.href);
+const bc = new BroadcastChannel("auth");
+const token1 = url.searchParams.get("token1");
```

Comment:
Same suggestion, can we give it a more meaningful name?

Reply:
I renamed it to login_token, as you receive the token through the URL that is saved in the localstorage after logging in (OAuth).